.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+If this is a general issue, you are encouraged to use the following guideline.
+
+#### Please paste your output of croutonversion from chroot below:
```

Comment:
@DennisLfromGA are we recommending croutonversion or just running `sudo edit-chroot -al`?